tests/integration/sample_server.cc

```diff
@@ -13,6 +14,11 @@ int main(int argc, char** argv) {
   // create an http server running on port 8080
   Exposer exposer{"127.0.0.1:8080"};
 
+  // create a push gateway
+  auto labels = Gateway::instance_label();
+
+  Gateway gateway{"127.0.0.1:9091", "sample_server", &labels};
```

Comment:
this will break the integration test run in CI, unless you also deploy a push gateway for the test.

Reply:
I split it into pull mode `sample_server`(as it was before) and push mode `sample_client`